pvacseq/lib/__init__.py

```diff
@@ -7,6 +7,7 @@
     "coverage_filter",
     "download_example_data",
     "generate_fasta",
+    "generate_protein_fasta",
```

Comment:
I didn't realize there was already a "generate_fasta" command.  Maybe "generate-annotated-fasta" would better differentiate between the intended uses of the two?